README.md

```diff
@@ -1,92 +1,150 @@
+- [Github sources](https://github.com/strawberrymusicplayer/strawberry)
+- [Github MSVC](https://github.com/strawberrymusicplayer/strawberry-msvc)
+- [Github MSVC prebuilt binary dependencies](https://github.com/strawberrymusicplayer/strawberry-msvc-dependencies)
+
+## Manual installation on Windows
+
+Download and install manually:
+- [Git](https://git-scm.com/downloads)
+    - Default settings can be used
+- [Visual Studio 2022 Community](https://visualstudio.microsoft.com/vs/)
+    - Select `Desktop development with C++`
+
+## Windows setup
+
+Open a Windows Terminal and type:
+
+```
+mkdir C:\data\projects\strawberry
+cd C:\data\projects\strawberry
+git clone https://github.com/strawberrymusicplayer/strawberry-msvc.git
+cd strawberry-msvc
+.\download.bat
+```
+
+Install manually from `C:\data\projects\strawberry\msvc_\downloads`:
+- **Required**:
+    - Install `7z<VERSION>-x64.exe`
+    - Install `cmake-<VERSION>-windows-x86_64.msi`
+    - Install `python-<VERSION>-amd64.exe`
+        - In first dialog:
+            - Check: `Add python.exe to PATH`
+            - Click `Install Now` (to use defaults)
+        - In dialog `Setup was successful`:
+            - Click: `Disable path length limit`
+    - Install `strawberry-perl-<VERSION>-64bit.msi`
+    - Extract `win_flex_bison-<VERSION>.zip`
+- **Recommended: When building dependencies from source**:
+    - Install `nasm-<VERSION>-installer-x64.exe`
+        - Run as normal user and click OK 
+        - Install in default directory `C:\Users\<USER>\AppData\Local\bin\NASM`
+- **Recommended: When creating NSIS Windows installer**:
+    - Install NSIS: `nsis-<NSIS_VERSION>-setup.exe`
+    - Extract NSIS `LockedList.zip` plugin:
+        - Copy `LockedList\Plugins\LockedList64.dll` to `C:\Program Files (x86)\NSIS\Plugins\LockedList64.dll`
+        - Copy `LockedList\Plugins\x86-unicode\LockedList.dll` to `C:\Program Files (x86)\NSIS\Plugins\x86-unicode\LockedList.dll`
+    - Extract NSIS `Inetc.zip` plugin:
+        - Copy `Inetc\Plugins\x86-unicode\INetC.dll` to `C:\Program Files (x86)\NSIS\Plugins\x86-unicode\INetC.dll`
+    - Extract NSIS `Registry.zip` plugin:
+        - Copy `Registry\Desktop\Plugin\registry.dll` to `C:\Program Files (x86)\NSIS\Plugins\registry.dll`
+        - Copy `Registry\Desktop\Plugin\registry.dll` to `C:\Program Files (x86)\NSIS\Plugins\x86-unicode\registry.dll`
+
+Delete `pkg-config` files to prevent conflicts with Strawberry's own `pkg-config`:
+- Delete `C:\strawberry\perl\bin\pkg-config`
+- Delete `C:\strawberry\perl\bin\pkg-config.bat`
+
+Windows Settings | System | About | Advanced system settings | Tab Advanced | Environment Variables:
+- System variables | Path | Edit:
+    - Delete: `C:\Strawberry\c\bin` (To prevent conflicts with other utilities)
+    - Add: `C:\Program Files\Git\bin` (This is for `sed` and `patch` utilities)
+    - Add: `C:\Users\user\AppData\Local\bin\NASM`
+    - Add: `C:\data\projects\strawberry\msvc_\downloads\win_flex_bison-<VERSION>`
```

Comment:
Anything under the msvc_ directory is used by the bat files and shouldn't be used for permanent installation, I suggest to change this to `c:\win_flex_bison`, it's already done by install.bat

Reply:
Path `c:\win_flex_bison` is used by `build.bat`, so removed environment variable.

Note: The `install.bat` does not install Python, Meson, NASM and NSIS correctly and works only with a manual installation, so did not mention this script. Maybe for another MR to fix.

Path environment variable to `win_flex_bison` removed in 2c2b21d20ebdc7232f6079114a627052fd7f1cb3.